canvasapi/account.py

```diff
@@ -642,6 +642,26 @@ def get_all_outcome_links_in_context(self):
             "accounts/{}/outcome_group_links".format(self.id),
         )
 
+    def get_authentication_event(self, **kwargs):
```

Comment:
Since this method returns multiple events, it should be renamed to `get_authentication_events` (plural)